js/apps.js

```diff
@@ -17,42 +32,45 @@ if (localStorage.getItem('bookArray') === null) {
   bookArray = JSON.parse(localStorage.getItem('bookArray'));
 }
 
-function createBook(title, author) {
-  const newBook = new Book(title, author);
-  bookArray.push(newBook);
-  localStorage.setItem('bookArray', JSON.stringify(bookArray));
-}
-
+const bookOject = new Book();
 addBookButton.addEventListener('click', (event) => {
   event.preventDefault();
   const bookTitle = addBook.querySelector('input[name="title"]').value;
   const bookAuthor = addBook.querySelector('input[name="author"]').value;
-  createBook(bookTitle, bookAuthor);
-  bookList.innerHTML += `<li class="book">
-        <p class="title">${bookTitle}</p>
-        <p class="author">${bookAuthor}</p>
-        <button type="button" class="delete-btn">Remove</button>
-        <hr />
-      </li>`;
+  if (bookTitle.length > 1 && bookAuthor.length > 1) {
+    bookOject.createBook(bookTitle, bookAuthor);
+    bookOject.updateCollection();
+    bookList.innerHTML += `<li class="book">
+          <p>
+          "<span class="title">${bookTitle}</span>" by
+          <span class="author">${bookAuthor}</span>
+          </p>
+          <button type="button" class="delete-btn">Remove</button>
+        </li>`;
+  } else {
+    const div = document.createElement('div');
+    div.className = 'validation-error';
+    div.textContent = 'Please Book Title or Book Author Shouldn\'t be empty';
+    div.style.display = 'block';
+    addBook.appendChild(div);
+  }
 });
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a feature that cleans the form `inputs` field when you add a book to make your page more professional.